tests/test__collection_api.py

```diff
@@ -1532,6 +1532,45 @@ def test__aggregate_project_out(self):
         self.assertEqual(expect, new_actual)
         self.assertEqual(expect, list(old_actual))
 
+    def test__aggregate_project_include_in_exclusion(self):
+        self.db.collection.insert_one({'_id': 1, 'a': 2, 'b': 3})
+        with self.assertRaises(ValueError) as err:
+            self.db.collection.aggregate([
+                {'$match': {'_id': 1}},
+                {'$project': {
+                    'a': False,
```

Comment:
Note that the order of iteration is not fixed in Python, therefore your two tests might just do the same thing. If you really want to test your code, you can import collections and use an OrderedDict. I'm also fine keeping only one of those two tests, or just leave it like that if you're lazy ;-)